modules/primer-utilities/lib/typography.scss

```diff
@@ -221,7 +221,7 @@
 
 /* Set to monospace font */
 .text-mono {
-  font-family: $mono-font;
+  font-family: $mono-font !important;
```

Comment:
!important is needed in order for the monospace font to have effect with other marketing headers, e.g. `h00-mktg` when you want a big monospace header like on github.com/ten https://github.com/github/github/pull/102693#discussion_r236315756